jenkins/post-build-task2.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+set -e
+
+OLDIFS=$IFS
+ISSUE_TITLE="Jenkins Randomized Gradle Check Failed on ${sha1}"
+
+###############################################
+
+function create_issue() {
```

Comment:
This particular script was written in this way to make sure auto / random checks is running daily without adding a new workflow.

Also, this script is used based on the fact that ghprb is in action.
Have you considered how to setup ghprb on new Jenkins instances to reproduce the current setup?

And probably we should rewrite this as the old code is not in good shape but a temp fix solution.

Thanks.